src/core.md

```diff
@@ -25,10 +69,62 @@ graph
   datajoint/djlab --> datajoint/djlabhub;
 ```
 
-- **[datajoint/mysql](https://datajoint.com/docs/core/mysql-docker/)**: An optimized, MySQL backend for [data pipelines](../glossary#data-pipeline).
-- **[datajoint/miniconda3](https://datajoint.com/docs/core/miniconda3-docker/)**: A minimal Python image with [conda](https://docs.conda.io/en/latest/).
-- **[datajoint/djbase](https://datajoint.com/docs/core/djbase-docker/)**: Adds only dependencies for managing [data pipelines](../glossary#data-pipeline).
-- **[datajoint/djtest](https://datajoint.com/docs/core/djtest-docker/)**: Adds testing tools like [pytest](https://docs.pytest.org/en/7.1.x/).
-- **[datajoint/datajoint](https://datajoint.com/docs/core/datajoint-python/)**: Official image for managing [data pipelines](../glossary#data-pipeline).
-- **[datajoint/djlab](https://datajoint.com/docs/core/djlab-docker/)**: Adds a local [Jupyter Lab](https://jupyterlab.readthedocs.io/en/stable/) environment.
-- **[datajoint/djlabhub](https://datajoint.com/docs/core/djlabhub-docker/)**: Adds a client to allow hosting with [Jupyter Hub](https://jupyter.org/hub).
\ No newline at end of file
+<div class="grid cards" markdown>
+
+-   :fontawesome-brands-docker:{ .lg .middle } **datajoint/mysql**
+
+    ---
+
+    An optimized, MySQL backend for [data pipelines](../glossary#data-pipeline).
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/core/mysql-docker/)
+
+-   :fontawesome-brands-docker:{ .lg .middle } **datajoint/miniconda3**
+
+    ---
+
+    A minimal Python image with [conda](https://docs.conda.io/en/latest/).
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/core/miniconda3-docker/)
+
+-   :fontawesome-brands-docker:{ .lg .middle } **datajoint/djbase**
+
+    ---
+
+    Adds only dependencies for managing [data pipelines](../glossary#data-pipeline).
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/core/djbase-docker/)
+
+-   :fontawesome-brands-docker:{ .lg .middle } **datajoint/djtest**
+
+    ---
+
+    Adds testing tools like [pytest](https://docs.pytest.org/en/7.1.x/).
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/core/djtest-docker/)
+
+-   :fontawesome-brands-docker:{ .lg .middle } **datajoint/datajoint**
+
+    ---
+
+    Official image for managing [data pipelines](../glossary#data-pipeline).
+
+    [:octicons-arrow-right-24: Getting started](https://datajoint.com/docs/core/datajoint-python/)
```

Comment:
Should this link to the docs for that docker image?

Reply:
This is also intended.

The actual `Dockerfile` and repo used for the official DJ image is our `datajoint-python` repo though it is old and needs to be updated. This means we won't have a dedicated docs page for just the container image. We'll likely add a section within the DJ Python docs just for getting setup with docker. Perhaps we can just add it in the installation section [here](https://datajoint.com/docs/core/datajoint-python/latest/getting_started/).

Reply:
Thank you!